src/site/layouts/faq_multiple_q_a.drupal.liquid

```diff
@@ -0,0 +1,84 @@
+{% include "src/site/includes/header.html" with drupalTags = true %}
+
+{% include "src/site/includes/alerts.drupal.liquid" %}
+{% include "src/site/includes/preview-edit.drupal.liquid" %}
+{% include "src/site/includes/breadcrumbs.drupal.liquid" %}
+
+<div id="content" class="interior">
+  <main class="va-l-detail-page">
+    <div class="usa-grid usa-grid-full">
+      <div class="usa-width-three-fourths">
+        {% if !entityPublished %}
+          <div class="usa-alert usa-alert-info" >
+            <div class="usa-alert-body">
+              <p class="usa-alert-text">You are viewing a draft.</p>
+            </div>
+          </div>
+        {% endif %}
+        <article class="usa-content">
+          <h1>{{ title }}</h1>
+
+          <div class="va-introtext">
+            {{ fieldIntroTextLimitedHtml.processed }}
+          </div>
+
+          {% if fieldAlertSingle %}
+            {% include "src/site/paragraphs/alert_single.drupal.liquid" with entity = fieldAlertSingle.entity %}
+          {% endif %}
+
+          {% if fieldButtonsRepeat %}
+            <div class="vads-u-margin-y--3">
+              {% for fieldButton in fieldButtons  %}
+                {% include "src/site/paragraphs/button.drupal.liquid" with entity = fieldButton.entity %}
+              {% endfor %}
+            </div>
+          {% endif %}
+
+          {% if fieldTableOfContentsBoolean %}
+          <section id="table-of-contents">
+            <h2 class="vads-u-margin-bottom--2 vads-u-font-size--lg">On this page</h2>
+            <ul class="usa-unstyled-list"></ul>
+          </section>
+          {% endif %}
+
+          {% for fieldQAGroup in fieldQAGroups %}
+            <h2>{{ fieldQAGroup.entity.fieldSectionHeader }}</h2>
+            <div class="usa-accordion-bordered">
+              <ul class="usa-unstyled-list">
+                {% for fieldQA in fieldQAGroup.entity.fieldQAs %}
+                  <li>
+                    <button
+                      aria-controls="{{ fieldQA.entity.entityBundle }}-{{ fieldQA.entity.entityId }}"
+                      class="usa-accordion-button usa-button-unstyled"
+                      aria-expanded="false"
+                    >{{ fieldQA.entity.title }}</button>
```

Comment:
Even though this is an accordion, the data structure is too different to be able to make use of the collapsible panel paragraph template.